src/api/operator/utils.ts

```diff
@@ -49,3 +48,98 @@ export function getOrderStatus(order: RawOrder): OrderStatus {
     return 'open'
   }
 }
+
+/**
+ * Get order filled amount, both as raw amount (in atoms) and as percentage (from 0 to 1)
+ *
+ * @param order The order
+ */
+export function getOrderFilledAmount(order: RawOrder): { amount: BigNumber; percentage: BigNumber } {
```

Comment:
why is not asking you to add the return type? did we turn off the super strict mode?

Reply:
What do you mean? The return type is there :thinking: